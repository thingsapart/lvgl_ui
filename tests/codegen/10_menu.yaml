../../examples/10_menu.yaml