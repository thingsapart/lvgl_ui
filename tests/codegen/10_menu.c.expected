/* AUTO-GENERATED by the 'c_code' backend */

#include "lvgl.h"
#include "c_gen/lvgl_dispatch.h" // For obj_registry_add
#include "data_binding.h"

void create_ui(lv_obj_t* parent) {
    // unnamed: dropdown_0 (dropdown)
    lv_obj_t* dropdown_0 = lv_dropdown_create(parent);

    lv_obj_align(dropdown_0, LV_ALIGN_TOP_LEFT, 10, 10);
    lv_dropdown_set_options(dropdown_0, "New project\nNew file\nSave\nSave as ...\nOpen project\nRecent projects\nPreferences\nExit");
    lv_dropdown_set_text(dropdown_0, "Menu");
    lv_dropdown_set_selected_highlight(dropdown_0, false);

}
